src/github/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@modelcontextprotocol/server-github",
-  "version": "0.5.1",
+  "version": "0.6.0",
```

Comment:
At the moment, all server versions have to be bumped in tandem. Can you please revert this or bump all servers?